build/packer/setup_filebeat_on_teamcity_agent.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export FILEBEAT_CONFIG_PATH=/tmp/filebeat.yml
+
+# Install filebeat service and run at startup
+echo "Installing Filebeat..."
+# Should be latest version of Filebeat (not hard requirement)
+curl -L -o filebeat-amd64.deb https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-7.7.1-amd64.deb
```

Comment:
Why are we doing this with curl and not APT/YUM?

https://www.elastic.co/guide/en/beats/filebeat/current/setup-repositories.html

Reply:
Will test and fix.

Reply:
This should also allow us to install the latest.